### DISCUSSION TITLE
Bayesian estimation of non-Gaussian SVARs help to understand this code

### DISCUSSION THREAD

**Discussion by Umar04R** [community/community#143451] [Programming Help]
### Select Topic Area

Question

### Body

help me to understand package for Bayesian estimation of non-Gaussian SVARs, see [bsvar](https://github.com/jetroant/bsvar).

This package ([rbsvar](https://github.com/jetroant/rbsvar)) implements all the methods discussed in [Anttonen, Lanne and Luoto (2023)](https://onlinelibrary.wiley.com/doi/full/10.1002/jae.3019).

rbsvar

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @Umar04R ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
